libbeat/template/processor.go

```diff
@@ -488,5 +488,17 @@ func getDefaultProperties(f *mapping.Field) common.MapStr {
 	if f.CopyTo != "" {
 		properties["copy_to"] = f.CopyTo
 	}
+
+	if f.MetricType != "" || f.Unit != "" {
```

Comment:
We should check for the ES version here, as meta was added in 7.6, so I believe this would break talking to any ES <= 7.5. We do version checks in some other parts of this file, it should be easy to skip this if the ES doesn't support meta